components/onboarding/steps/program-duration-step.tsx

```
@@ -0,0 +1,233 @@
+    return (
+        <motion.div
+            initial={{ opacity: 0, y: 20 }}
+            animate={{ opacity: 1, y: 0 }}
+            exit={{ opacity: 0, y: -20 }}
+            transition={{ duration: 0.3 }}
+            className="space-y-6"
+        >
+            <div className="text-center">
+                <h2 className="text-2xl font-bold mb-2">Program Details</h2>
+                <p className="text-muted-foreground">
+                    Set the duration and details for {data.client?.name}'s program
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)